config/locales/diaspora/en.yml

```diff
@@ -1008,8 +1008,13 @@ en:
 
   profiles:
     edit:
-      your_public_profile: "Your public profile"
-      your_private_profile: "Your private profile"
+      basic: "Basic"
+      additional: "Additional"
```

Comment:
I'm actually not sure about the wording here, "Additional" on its own feels a bit cut off, but also "Additional profile data" feels a bit technical. @goobertron any ideas?